api/src/main/java/cz/metacentrum/perun/spRegistration/rest/controllers/common/ConfigController.java

```diff
@@ -10,10 +10,7 @@
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.bind.annotation.SessionAttribute;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Do not use asterisk wildcard. Rather import each dependency separately. (Probably problem in IDE settings)

Reply:
Done